packages/flutter_tools/test/integration.shard/migrate_status_test.dart

```diff
@@ -0,0 +1,202 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
can you move this file to `packages/flutter_tools/test/command.shard/permeable/migrate_status_test.dart`?